### DISCUSSION TITLE
Is there a db / dump of NPC models?

### DISCUSSION THREAD

**Discussion by krateng** [Jonathan-Greve/GuildWarsMapBrowser#18] [General]
Not directly related to the software, but I'm having a hard time finding leads so figured I'd ask here. Since the GW Wiki render project has extracted lots of NPC models to create rendered images of them, are the actual 3D models also available somewhere? Mainly Heroes, Henchmen and story NPCs. I can't use this program because I'm on Linux, plus tracking down all the NPCs seems like a lot of work that would be silly to repeat if it has already been done.

**Comment by Jonathan-Greve** [comment]
Unfortunately I don't think such a project exists. If you could run my program here, you could use _GWToolbox_ to get the `File Id` of the NPCs you wanted by targeting the NPC and using the _GWToolbox_ `info window`. But even then if you want all NPCs it would be a long process.

**Reply by krateng** [reply]
Hm, that would be some progress, I can use your program with wine well enough to at least export from known IDs I think. Are all NPCs just one model? I read something along the lines of NPCs consisting of multiple meshes that are combined in some proprietary fashion that would have to manually be replicated?

**Reply by Jonathan-Greve** [reply]
I think almost all NPCs are just a single model. I think it's mostly the player models that are made up of individual meshes.
